README.md

```diff
@@ -14,17 +14,16 @@ impress.js may not help you if you have nothing interesting to say ;)
 HOW TO USE IT
 ---------------
 
-[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke ;)
+[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke!
```

Comment:
I am thinking about removing this, given that we received some useful feedback in [Hacker News](https://news.ycombinator.com/item?id=12027169) recently:

> I thought this project was really cool when I first came across it, but there was one thing that really offended me about it and made me refuse to ever use it:

> >How to Use It

> >Use the source, Luke ;)

> >Please note that impress.js was created for developers. Some basic knowledge of html, css and javascript is a prerequisite to install and use its features.

> So, in other words, "please, reverse engineer my work because I'm too lazy to document my code."

> Sure, they've put in a lot of comments, but there's still no clearly defined interfaces or guides for use.

> It's all comments like 'delegated handler for clicking on step elements' and big ascii art of yoda.
Documentation is not just for _newbs_. It's an important part of making a third party library. If you want other developers to use your code, then make it easy to use. To not do so is incredibly user-hostile, and it's not going to get you anywhere.

> Could you imagine if people had attitudes like this in other fields of engineering? "Here's your new car." "Thanks, how do I change a tire or put in gas? Does it have a manual?" "Manual? Just read the schematics. If you know anything about mechanical engineering or combustion, you should be able to figure it out!"

@bartaz does it make sense changing this?